repository-data/application/src/main/resources/hcm-config/namespaces/hee/publicationLandingPage.yaml

```diff
@@ -289,38 +285,38 @@ definitions:
             /cluster.options:
               jcr:primaryType: frontend:pluginconfig
               nodetypes: ['hee:logoGroup']
-          /publicationProfessions:
+          /globalTaxonomyProfessions:
             jcr:primaryType: frontend:plugin
             caption: Professions
-            field: publicationProfessions
-            multiselect.type: palette
-            palette.alloworder: false
-            palette.maxrows: '10'
-            plugin.class: org.onehippo.forge.selection.frontend.plugin.DynamicMultiSelectPlugin
-            selectlist.maxrows: '10'
-            valuelist.provider: service.valuelist.default
+            field: globalTaxonomyProfessions
+            hint: Choose at least one relevant profession and/or healthcare topic
```

Comment:
Since the hint is specific to a field, can we stick to the hint similar to the previous one if possible ?

Reply:
Hint has been st as per spec: https://hee-tis.atlassian.net/wiki/spaces/NWPK/pages/3611754507/Publication+Landing+Page

Reply:
Sorry, I didn't check the spec, thanks 👍